schema-next.yaml

```diff
@@ -2,6 +2,14 @@ file_format: 1.1.0
 schema_url: https://opentelemetry.io/schemas/next
 versions:
   next:
+    metrics:
```

Comment:
since the attribute is renamed and the old one is deprecated, we should apply this change to all signals, not just specific metric